packages/lexical-table/src/LexicalTableNode.ts

```diff
@@ -83,19 +117,21 @@ export class TableNode extends ElementNode {
   }
 
   static importJSON(serializedNode: SerializedTableNode): TableNode {
-    const tableNode = $createTableNode();
+    const tableNode = $createTableNode(serializedNode.colWidths);
     tableNode.__rowStriping = serializedNode.rowStriping || false;
     return tableNode;
   }
 
-  constructor(key?: NodeKey) {
+  constructor(colWidths?: number[], key?: NodeKey) {
     super(key);
     this.__rowStriping = false;
+    this.__colWidths = colWidths;
   }
```

Comment:
For compatibility it would be better to leave this out of the constructor and put it in `afterCloneFrom`.

```suggestion
  constructor(key?: NodeKey) {
    super(key);
    this.__rowStriping = false;
  }
```